AWS/AccessKeyRotatedWithin180days.json

```diff
@@ -0,0 +1,72 @@
+{
+  "configuration": {
+    "name": "Custom Rule - Check if IAM User access key rotated within 180 days",
+    "description": "Check if IAM User access key rotated within 180 days",
+    "service": "IAM",
+    "resourceType": "iam-user",
+    "severity": "HIGH",
+    "enabled": true,
+    "provider": "aws",
+    "categories": [
+      "security"
+    ],
+    "remediationNotes": "Rotate access key",
+    "attributes": [
+      {
+        "name": "key_create_date",
+        "path": "data.AccessKeyMetadata[0].CreateDate",
+        "required": true
+      }
+    ],
+    "rules": [
+      {
+        "conditions": {
+          "any": [
+            {
+              "fact": "key_create_date",
+              "operator": "dateComparison",
+              "value": {
+                "days": 180,
+                "operator": "within"
+              }
+            }
+          ]
+        },
+        "event": {
+          "type": "Check if IAM User access key rotated within 180 days"
+        }
+      }
+    ]
+  },
+  "resource": {
+    "accountId": "933d1234-1234-1234-1234-1234ce55ce62",
+    "organisationId": "12345234-6789-6789-6789-2345ce557890",
+    "resourceId": "1234123412349999234",
+    "service": "IAM",
+    "ccrn": "ccrn:aws:933d1234-1234-1234-1234-1234ce55ce62:IAM:global:user-1234123412349999234",
+    "region": "global",
+    "descriptorType": "iam-user",
+    "data": {
```

Comment:
@josh-wu 
There is no resource.data.resourceId so the dry run result message will have undefined string, so maybe also add the resourceId as well
![image](https://github.com/cloudconformity/custom-rule-examples/assets/58759429/0a36d156-65cc-4382-bb51-e77ccc535de0)

Reply:
Nice catch 🚀